opm/autodiff/BlackoilPropsAdFromDeck.cpp

```diff
@@ -40,6 +40,18 @@
 #include <opm/parser/eclipse/Deck/Deck.hpp>
 #include <opm/parser/eclipse/EclipseState/EclipseState.hpp>
 
+
+namespace {
+    inline
+    void silenceCompilerWarnings() {
```

Comment:
An explanation as to why we see warnings occasionally for un-successful builds would be nice, but I do not have one. Given that successful builds do not produce a warning, I think it is best to remove this.